tests/unit/conftest.py

```diff
@@ -0,0 +1,265 @@
+import paddle
+import pytest
+import tensorflow as tf
+import torch
+
+
+@pytest.fixture
+def torch_dense_model():
+    return torch.nn.Sequential(
+        torch.nn.Linear(in_features=128, out_features=128),
+        torch.nn.ReLU(),
+        torch.nn.Linear(in_features=128, out_features=64),
+        torch.nn.ReLU(),
+        torch.nn.Linear(in_features=64, out_features=32),
+        torch.nn.ReLU(),
+        torch.nn.Linear(in_features=32, out_features=10),
+        torch.nn.Softmax(),
```

Comment:
What are we trying to acheive by having so many layers in a test model?

Reply:
reuse of the tailor fixtures to make sure different models works when attach projection head. @tadejsv